invenio_rdm_records/resources/serializers/csl/schema.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""CSL based Schema for Invenio RDM Records."""
+
+from invenio_access.permissions import system_identity
+from invenio_vocabularies.proxies import current_service as vocabulary_service
+from marshmallow import Schema, fields, missing, post_dump
+from marshmallow_utils.fields import SanitizedUnicode
+
+
+class PersonOrOrgSchemaCSL(Schema):
+    """Creator/contributor common schema for v4."""
```

Comment:
I guess v4 comes from the datacite schema and it shouldn't be here?

```suggestion
    """Creator/contributor common schema."""
```